cloud-core/src/main/java/cloud/commandframework/Command.java

```diff
@@ -159,7 +159,7 @@ public Command(
      */
     @Deprecated
     public Command(
-            final @NonNull Map<@NonNull CommandArgument<C, ?>, @NonNull Description> commandArguments,
+            final @NonNull Map<@NonNull CommandArgument<C, ?>, @NonNull Description<C>> commandArguments,
```

Comment:
every type variable for these should be `<? super C>` -- that makes things more flexible

Reply:
I was following the existing patterns with `CommandArgument<C, ?>` here. Would it be preferred to change the method signature to `final @NonNull Map<@NonNull CommandArgument<? super C, ?>, @NonNull Description<? super C>>` ?